lib/perl/Genome/Model/Tools/Picard/FilterSamReads.pm

```diff
@@ -53,51 +58,18 @@ sub help_detail {
 EOS
 }
 
-sub execute {
-    my $self = shift;
-
-    my $input_file = $self->input_file;
-    my $output_file = $self->output_file;
-
-    my $jar_path = $self->picard_path . '/FilterSamReads.jar';
-    unless ( -e $jar_path ) {
-        die( "Failed to find $jar_path!" .
-          "This command may not be available in version " . $self->use_version );
-    }
+sub minimum_version_required { '1.77'; }
+sub _jar_name { 'FilterSamReads.jar'; }
+sub _java_class { 'net.sf.picard.sam.FilterSamReads'; }
```

Comment:
The most recent commit should fix this not only in FilterSamReads but other occurrences in the picard tree.